### PR TITLE
Docs by Tom

### PR DESCRIPTION
**Pull request by tomrudnick**

### CODE REVIEW

**Review by botrufus — APPROVED** [review]

File: docs/system/data-management.rst [hunk 1]
```diff
@@ -1,6 +1,24 @@
...
+
+The app stores the selected Course ID in the UserDefaults at **"shownCourseIDKey"** in order to always show the last selected
+course.
+The Artemis-Server URL is also stored in the UserDefaults at **"serverURL"**.
+
+****
+ThemisML
+****
+
 (Token, font size, user defaults, ML part data → 2 sections for PDM)
```
Comment (huber-florian) [inline docs/system/data-management.rst]: Please remove the default text.

File: docs/admin/third-party.rst [hunk 2]
```diff
@@ -1,4 +1,61 @@
...
+------------
+
+The Swift-Markdown-ui library is used to parse and display markdown content in a SwiftUI app. 
+This library is used in the Themis app to display markdown content from Artemis in a clean and formatted manner.
+
+Library repository: https://github.com/gonzalezreal/swift-markdown-ui (2.0.0 - next major)
+
+
+
 by iOS Tom Rudnick, Rearranging Files Katjana Kosic, ML part Andreas Cselovszky Paul Schwind, fastAPI, all imports
```
Comment (huber-florian) [inline docs/admin/third-party.rst]: Here as well please

**Review by huber-florian — APPROVED** [review]
Apart from that it looks very good!

File: docs/system/data-management.rst (hunk 1 above)
Comment (tomrudnick) [reply docs/system/data-management.rst]: removed

File: docs/admin/third-party.rst (hunk 2 above)
Comment (tomrudnick) [reply docs/admin/third-party.rst]: removed

**Pull request merged**